cocos/asset/assets/render-texture.ts

```diff
@@ -53,6 +53,10 @@ const _windowInfo: IRenderWindowInfo = {
     renderPassInfo: passInfo,
 };
 
+function isWebGPU (): boolean {
```

Comment:
Why need to do special check for webgpu? How about metal and vulkan?

Reply:
In WebGPU, it is possible to force the format to RGBA8, but the official recommendation is to call the function navigator.gpu.getPreferredCanvasFormat() to use the most suitable format for the swapchain. This function may return BGRA8 on some devices, so it is necessary to make a distinction.

Reply:
But why not just get format from swapchain for all platforms?